zhaquirks/develco/io_module.py

```diff
@@ -0,0 +1,53 @@
+"""Develco IO Module."""
+
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.zcl.clusters.general import BinaryInput, OnOff
+
+(
+    QuirkBuilder("frient A/S", "IOMZB-110")
+    # Name the two outputs
+    .change_entity_metadata(
+        endpoint_id=116,
+        cluster_id=OnOff.cluster_id,
+        unique_id_suffix="116-6",
+        new_fallback_name="COM 1",
+        new_translation_key="frient_com1",
+    )
+    .change_entity_metadata(
+        endpoint_id=117,
+        cluster_id=OnOff.cluster_id,
+        unique_id_suffix="117-6",
+        new_fallback_name="COM 2",
+        new_translation_key="frient_com2",
+    )
+    # And the two inputs
+    .change_entity_metadata(
+        endpoint_id=112,
+        cluster_id=BinaryInput.cluster_id,
+        unique_id_suffix="112-15",
+        new_fallback_name="IN1",
```

Comment:
Is this supposed to be "IN1" or should it be "IN 1 (and so on for the others)?

Reply:
It's intentional, the device itself has labels for these pins and uses this exact format: `IN1` and `COM 1`.